pkg/server/service.go

```diff
@@ -129,11 +134,13 @@ func NewCRIContainerdService(config criconfig.Config, client *containerd.Client)
 	c.imageFSPath = imageFSPath(config.ContainerdRootDir, config.ContainerdConfig.Snapshotter)
 	logrus.Infof("Get image filesystem path %q", c.imageFSPath)
 
-	c.netPlugin, err = ocicni.InitCNI(config.NetworkPluginConfDir, config.NetworkPluginBinDir)
-	if err != nil {
-		return nil, fmt.Errorf("failed to initialize cni plugin: %v", err)
+	c.netPlugin, err = cni.New(cni.WithMinNetworkCount(2),
```

Comment:
Can we make `2` a const, and explain why it is 2?

And are we sure that for different CNI plugins, only 2 are needed?

Reply:
2 is - 1 for loop back and 1 for default network . You ar right about const. Willl do